core/primitives/chain/message.hpp

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_PRIMITIVES_CHAIN_MESSAGE_HPP
+#define CPP_FILECOIN_CORE_PRIMITIVES_CHAIN_MESSAGE_HPP
+
+#include "common/buffer.hpp"
+#include "primitives/address/address.hpp"
+#include "primitives/big_int.hpp"
+
+// TODO (yuraz) not really used yet, just a sketch remove this message after
```

Comment:
could you create a task or make a reference to the Message task?

Reply:
task FIL-72 already exists, I thought I need this structure before I finally decomposed the task, but it is required by FIL-72, so I don't want to remove it wasting time.